README.md

```diff
@@ -43,41 +44,130 @@ Check the [Troubleshooting section](#troubleshooting) if something goes wrong wi
+ 
+ ```node . create --config config/create.remote.sample-AWS.json --verbose``` 
+ 
+The process will start creating an instance of polkadot on AWS. The process with create a 2 node cluster using `m4.large` machines. An IAM role and a VPC will be created that will contain the Amazon EKS for the deployment along with the required security groups and ingress rules. You may review the entire process [here](https://github.com/w3f/polkadot-deployer/tree/master/terraform/aws).
+
+If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
+
+ ```node . destroy aws-testnet```  
+</p>
+</details>
+
+<details><summary>Azure</summary>
+<p>
+To deploy polkadot on Azure you're required to set your credentials. You can do this by following this 
+[documentation](https://docs.microsoft.com/en-us/azure/app-service/deploy-configure-credentials).
+
+In order to deploy polkadot on Azure you can use a the preset configuration file: ```create.remote.sample-AZURE.json``` and issue the following command:  
+
+ ```node . create --config config/create.remote.sample-AZURE.json --verbose```  
+
+The process will start creating an instance of polkadot on Azure, deployed as a 2 ```Standard_D2s_v3```  node kubernetes cluster on your default location connected through a virtual network. Furthermore the required security groups and inbound rules will be applied to your deployment. You may review the entire process [here](https://github.com/w3f/polkadot-deployer/tree/master/terraform/azure). 
+
+If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
+
+ ```node . destroy azure-testnet```  
+</p>
+</details>
+
+<details><summary>Digital Ocean</summary>
+<p>
+To make a deployment on Digital Ocean you're required to configure your Digital Ocean's credentials. You can do this by setting the `DIGITALOCEAN_ACCESS_TOKEN` environment variable. You can get your access token by following the 
+[documentation](https://www.digitalocean.com/docs/api/create-personal-access-token/).
+
+In order to deploy polkadot on Digital Ocean you can use the preset configuration file: ```create.remote.sample-DO.json``` and issue the following command:  
+
+ ```node . create --config config/create.remote.sample-DO.json --verbose```  
+
+The process will start creating an instance of polkadot on Digital Ocean, using a 2 node kubernetes cluster of `s-4vcpu-8gb` machines. You may review the entire process [here](https://github.com/w3f/polkadot-deployer/tree/master/terraform/do).
+
+If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
+
+ ```node . destroy do-testnet```
+</p>
+</details>
+
+### Multi provider deployment
+You may also wish to run a multi AZ multi provider deployments. In order to do so you can use a the preset configuration file: create.remote.sample.json and issue the following command:  
```

Comment:
We should make clear that the providers can be combined as desired, no need to use these three in this specific order.

Reply:
You may also wish to run a multi AZ multi-provider deployment. In order to do so, you can create a configuration file based on your requirements and create your deployment from there. Keep in mind that you can use any combination of these providers as you see fit. The configuration file: create.remote.sample.json exists only for the purpose of the tutorial and as an example of what you can do.

I think we should be ok.

Reply:
Yep sounds good :+1: